src/challenges/defi-challenges/simple-lending.mdx

```diff
@@ -0,0 +1,244 @@
+---
+name: Simple Lending
+index: 3
+level: 3
+website: "https://github.com/Turupawn/LendingChallengeLevelUp"
+summary: Understand the inner workings of Aave by completing the Lending challenge.
+heroImage: "/images/level-up-preview.png"
+labels: ["DeFi"]
+---
+
+import ContentCopy from "src/components/ContentCopy";
```

Comment:
In this new update, ContentCopy integration is no longer required. You can directly write code in traditional markdown format.